code/core/src/toolbar/components/ToolbarMenuSelect.tsx

```diff
@@ -40,16 +40,12 @@ export const ToolbarMenuSelect: FC<ToolbarMenuSelectProps> = withKeyboardCycle(
     const currentValue = globals[id];
     const isOverridden = id in storyGlobals;
     let icon = _icon;
-    let title = _title;
+    const title = _title;
```

Comment:
You no longer need to rename the `title` prop to `_title` and then create a new `title`. Could you please tidy that up?